flink-connector-bigquery-common/src/main/java/com/google/cloud/flink/bigquery/common/utils/BigQueryPartitionUtils.java

```diff
@@ -264,7 +264,7 @@ static String timestampRestrictionFromPartitionType(
                     // extract a datetime from the value and restrict
                     // between previous and next hour
```

Comment:
I know support for unbounded read is going to be removed. But this was a very simple and easy change to fix reading of Time Partitions based on DAY, MONTH and YEAR.